runtime/src/lib.rs

```diff
@@ -343,52 +343,56 @@ cfg_if::cfg_if! {
             fn contains(call: &Call) -> bool {
                 match call {
                     // Allowed calls:
-                    Call::System(_) | Call::Grandpa(_) | Call::Sudo(_) | Call::Timestamp(_) => true,
-
-                    // Prohibited calls:
-                    Call::Balances(_)
-                    | Call::Treasury(_)
-                    | Call::AdvisoryCommittee(_)
+                    Call::AdvisoryCommittee(_)
                     | Call::AdvisoryCommitteeMembership(_)
+                    | Call::Balances(_)
                     | Call::Council(_)
                     | Call::CouncilMembership(_)
-                    | Call::TechnicalCommittee(_)
-                    | Call::TechnicalCommitteeMembership(_)
-                    | Call::MultiSig(_)
+                    | Call::Currency(_)
                     | Call::Democracy(_)
-                    | Call::Scheduler(_)
-                    | Call::Preimage(_)
+                    | Call::Grandpa(_)
                     | Call::Identity(_)
-                    | Call::Utility(_)
+                    | Call::MultiSig(_)
+                    | Call::Preimage(_)
                     | Call::Proxy(_)
-                    | Call::Currency(_)
+                    | Call::Scheduler(_)
+                    | Call::Sudo(_)
+                    | Call::System(_)
+                    | Call::TechnicalCommittee(_)
+                    | Call::TechnicalCommitteeMembership(_)
+                    | Call::Timestamp(_)
+                    | Call::Treasury(_)
+                    | Call::Utility(_)
+                    | Call::Vesting(_) => true,
```

Comment:
```suggestion
                    | Call::Vesting(_) => {
                        return true;
                    },
```
Just visual sugar. First time I saw it, I couldn't really match it.